src/js/views/globals/patient-modal/patient-modal_views.js

```diff
@@ -58,28 +63,43 @@ const InputView = View.extend({
 });
 
 const SaveView = View.extend({
-  className: 'add-patient__save',
+  className: 'patient-modal__save',
   template: hbs`
-    <button class="button--green js-save" {{#if isDisabled}}disabled{{/if}}>{{ @intl.globals.addPatient.addPatientViews.saveView.saveBtn }}</button>
-    <button class="button--text u-margin--r-4 js-cancel">{{ @intl.globals.addPatient.addPatientViews.saveView.cancelBtn }}</button>
+    <button class="button--green js-save" {{#if isDisabled}}disabled{{/if}}>{{ @intl.globals.patientModal.patientModalViews.saveView.saveBtn }}</button>
+    <button class="button--text u-margin--r-4 js-cancel">{{ @intl.globals.patientModal.patientModalViews.saveView.cancelBtn }}</button>
   `,
   triggers: {
     'click .js-cancel': 'cancel',
     'click .js-save': 'click:save',
   },
+  modelEvents: {
+    'change': 'render',
+  },
   templateContext() {
     const state = this.getOption('state');
+    const isDisabled = this.isSubmitting || !this.model.hasChanged() || this.model.hasChanged() && !this.model.isValid({ preSave: true });
```

Comment:
I think the `this.model.hasChanged() &&` might not be needed?  you're only ever getting to that or if it has changed because of the previous or